lib/auth/apiserver.go

```diff
@@ -621,6 +621,9 @@ type upsertCertAuthorityRawReq struct {
 	TTL time.Duration   `json:"ttl"`
 }
 
+// upsertCertAuthority creates or updates a cert authority.
+// Deprecated: Use teleport.v1.trust.TrustService/UpsertCertAuthority instead.
+// DELETE IN 14.0.0
```

Comment:
Should all these removed methods be maintained with a handler that returns some "unimplemented" error rather than removed? I think we can't distinguish a `NotFound` due to the handler not existing from a `NotFound` because the resource itself doesn't exist.

For instance, creating a token with a v9 tctl on a v11 cluster results in "ERROR: object not found" rather than some better error.

Reply:
We should solve this at the apiserver level, by setting the default handler (or the handler for `/webapi/` in the proxy http server) to return 501 Not Implemented rather than 404 Not Found.